packages/tools/generator-widget/README.md

```diff
@@ -13,73 +13,60 @@
 
 This generator uses the Yeoman scaffolding tool to let you quickly create a [Mendix Pluggable Widget](https://docs.mendix.com/howto/extensibility/pluggable-widgets).
 
----
-
 ## Installation
 
-First, install [Yeoman](http://yeoman.io) and @mendix/widget using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+1. Install [node.js](https://nodejs.org/) (version >= 12).
+1. Make sure that you have an [NPM](https://www.npmjs.com/package/npm) package below version 7 installed. We advise to use the latest v6 release which can be installed with:
 
-```bash
-npm install -g yo
-npm install -g @mendix/generator-widget
-```
+    ```bash
+     npm install -g npm@latest-6
+    ```
 
----
+1. Install [Yeoman](http://yeoman.io):
 
-Then generate your new project inside an empty folder:
+    ```bash
+    npm install -g yo
+    ```
 
-```bash
-yo @mendix/widget
-```
+1. Install Pluggable Widgets Generator:
 
-or automatically create the folder using:
+    ```bash
+    npm install -g @mendix/generator-widget
+    ```
 
-```bash
-yo @mendix/widget widget name
-```
+## Scaffold a widget project
 
-## Scaffold a widget
+1. Generate your new project inside an empty folder:
 
-### 1. Provide the following information about your widget:
+    ```bash
+    yo @mendix/widget
+    ```
 
-The following information needs to be provided about your widget:
+    or automatically create the folder using:
 
--   name
--   description
--   organization
--   copyright
--   license
--   version
--   author
--   Mendix Project path
--   programming language
--   platform
+    ```bash
+    yo @mendix/widget MyWidgetName
+    ```
 
-Press <Enter> if you want to skip and use default values.
+    Note that `MyWidgetName` can consist of space characters as well.
 
-### 2.1. Using the task runner
+1. Provide the following information about your widget project (press <Enter> if you want to skip and use the default values):
 
-The widget generator will include the necessary files and tasks to your package.json for running the tasks over The Pluggable Widgets Tools.
+    - Widget name
+    - Description
+    - Organization
+    - Copyright
+    - License
+    - Version
+    - Author
+    - Mendix project path
+    - Programming language
+    - Platform
+    - Template
+    - Add unit tests
+    - Add end to end tests
 
-If necessary you can run the tasks using the commands
-
-```bash
-npm start
-```
-
-```bash
-npm run build
-```
-
-```bash
-npm run release
-```
-
----
-
-For more information, visit our [Mendix Pluggable Widget Tools repository](https://github.com/mendix/widgets-resources/tree/master/packages/tools/pluggable-widgets-tools).
-
-### 2.2. Which template do you want to use for the widget?
+### Template
```

Comment:
The previous heading had a description in the title to indicate the purpose of this section, which is now lost. I'm not a fan of putting it in the title, but could you either make the header a bit more descriptive or include a description under the header to indicate the purpose of this section? Right now it's pretty vague imo